kwc-challenge-card.html

```diff
@@ -9,26 +13,134 @@
 
 <dom-module id="kwc-challenge-card">
   <template>
+
     <style>
-      :host {
-        display: block;
-      }
+        :host {
+            @apply --layout-vertical;
+            border-radius: 3px;
+            background-color: white;
+            overflow: hidden;
+            transform: translateZ(0);
+        }
+        :host .kwc-cc {
+          width: var(--kwc-cc-width, 184px);
+          margin: auto;
+          cursor: pointer;
+        }
+        :host .title,
+        .card-content {
+            color: var(--color-black);
+            margin: 0;
+            font-size: 12px;
+            text-decoration: none;
+        }
+        :host .title {
+            font-size: 16px;
+            font-weight: bold;
+        }
+        :host .cover,
+        :host .cover > iron-image {
+            background-color: inherit;
+            width: 100%;
+            z-index: 0;
+            height: var(--kwc-cc-cover-height, 176px);
+        }
+        :host .cover {
+            position: relative;
+        }
+        :host .info {
+            @apply --layout-flex;
+            border-width: 0 1px 1px 1px;
+            border-style: solid;
+            border-color: var(--color-grey-lighter);
```

Comment:
I think this may have been a mistake in the original component – `--color-grey-lighter` is from an old set of colours. Probably best to use from from the new set of colours like this one: https://github.com/KanoComputing/web-components/blob/master/kano-style/color.html#L14